
const getData= async()=>
{
    try{
        const response= await
    }
    catch(err)
    {
        console.log(err);
        throw err;
    }
}

constGetAllUnApprovedAppointmenr=()=>
{
    return(

    )
}